server/src/main/java/org/opensearch/action/admin/cluster/decommission/awareness/put/DecommissionRequest.java

```diff
@@ -90,8 +94,16 @@ public DecommissionAttribute getDecommissionAttribute() {
         return this.decommissionAttribute;
     }
 
-    public TimeValue getDrainingTimeout() {
-        return this.drainingTimeout;
+    public TimeValue getDelayTimeout() {
+        return this.delayTimeout;
+    }
+
+    public void setNoDelay(boolean noDelay) {
+        this.noDelay = noDelay;
+    }
+
```

Comment:
we need to guarantee only one of no_delay and delay timeout is set

Reply:
We have done that in DecommissionRequest.